src/meltano/cli/compile.py

```diff
@@ -60,13 +60,12 @@ def compile_command(
     lint: bool,
     indent: int,
 ) -> None:
-    """
-    Compile a Meltano project into environment-specific manifest files.
+    """Compile a Meltano project into environment-specific manifest files.
```

Comment:
I think this causes some alignment problems:

### `main`

```
Usage: meltano add [OPTIONS] {extractor|extractors|loader|loaders|transform|tr
                   ansforms|orchestrator|orchestrators|transformer|transformer
                   s|file|files|utility|utilities|mapper|mappers|mapping|mappi
                   ngs} PLUGIN_NAME...

      Add a plugin to your project.

      Read more at https://docs.meltano.com/reference/command-line-interface#add
          

Options:
  --inherit-from TEXT  Add a plugin inheriting from an existing plugin in the
                       project or a discoverable plugin identified, by name.
  --variant TEXT       Add a specific (non-default) variant of the identified
                       discoverable plugin.
  --as TEXT            Shorthand for '--inherit-from', that can be used to add
                       a discoverable plugin to your project with a different
                       name. Usage:
                       
                       add <type> <inherit-from> --as <name>
  --from-ref TEXT      Reference a plugin definition to add from.
  --python TEXT        The Python version to use for the plugin. Only applies
                       to base plugins which have Python virtual environments,
                       rather than inherited plugins which use the virtual
                       environment of their base plugin, or plugins that run
                       in a container.
  --custom             Add a custom plugin. The command will prompt you for
                       the package's base plugin description metadata.
  --update             Update an existing plugin.
  --install            Install the subject plugin(s) automatically.
  --no-install         Do not install the subject plugin(s) automatically.
  --force-install      Ignore the required Python version declared by the
                       plugins.
  --database-uri TEXT  System database URI.
  --help               Show this message and exit.
```

### this branch

```
Usage: meltano add [OPTIONS] {extractor|extractors|loader|loaders|transform|tr
                   ansforms|orchestrator|orchestrators|transformer|transformer
                   s|file|files|utility|utilities|mapper|mappers|mapping|mappi
                   ngs} PLUGIN_NAME...

  Add a plugin to your project.

      Read more at https://docs.meltano.com/reference/command-line-interface#add
          

Options:
  --inherit-from TEXT  Add a plugin inheriting from an existing plugin in the
                       project or a discoverable plugin identified, by name.
  --variant TEXT       Add a specific (non-default) variant of the identified
                       discoverable plugin.
  --as TEXT            Shorthand for '--inherit-from', that can be used to add
                       a discoverable plugin to your project with a different
                       name. Usage:
                       
                       add <type> <inherit-from> --as <name>
  --from-ref TEXT      Reference a plugin definition to add from.
  --python TEXT        The Python version to use for the plugin. Only applies
                       to base plugins which have Python virtual environments,
                       rather than inherited plugins which use the virtual
                       environment of their base plugin, or plugins that run
                       in a container.
  --custom             Add a custom plugin. The command will prompt you for
                       the package's base plugin description metadata.
  --update             Update an existing plugin.
  --install            Install the subject plugin(s) automatically.
  --no-install         Do not install the subject plugin(s) automatically.
  --force-install      Ignore the required Python version declared by the
                       plugins.
  --database-uri TEXT  System database URI.
  --help               Show this message and exit.
```

Notice how the headline if misaligned with `Read more ...`.

I think the suggestions below fix that.

https://click.palletsprojects.com/en/8.1.x/documentation/#preventing-rewrapping

Reply:
Thank you for fixing this @edgarrmondragon

Reply:
No problem!